DTNL.LL.Logic/ProjectService.cs

```diff
@@ -38,6 +38,10 @@ public async Task UpdateAsync(int oldProjectId, Project newValues)
             if (newValues.CustomerName is not null) project.CustomerName = newValues.CustomerName;
             if (newValues.ProjectName is not null) project.ProjectName = newValues.ProjectName;
 
+            project.TimeRangeEnabled = newValues.TimeRangeEnabled;
+            if (newValues.TimeRangeStart is not null) project.TimeRangeStart = newValues.TimeRangeStart;
```

Comment:
If the user puts in null for the new time range, then it is not set, so that time range is not null. I thought it would be best for time range to never be null, and in the case that the user wants to put in a null, it actually keeps the old value. Should I make it set to null nonetheless?